src/lib/utils/create-expiration.ts

```diff
@@ -1,10 +1,22 @@
+interface IExpirationOptions {
+  /**
+   * The number of milliseconds to round the expiration date.
+   * Mostly used for testing.
+   */
+  rounding?: number;
```

Comment:
This is nitpicky, but does it make sense for rounding to be _any_ number, or a _limited set of numbers_ (e.g. 10, 100, 1000 ms)? Can typescript handle this, via 10 | 100 | 1000, etc? Alternatively: Might it make sense for the parameter to be a number corresponding to a place-value, or a precision, rather than a number of milliseconds? (Just offering food for thought; trusting you'll pick the approach that makes the most sense...)